### DISCUSSION TITLE
Is there a reason the calendar pop-up closes when going forward then back a month with `minValue` set?

### DISCUSSION THREAD

**Discussion by will-stone** [adobe/react-spectrum#3425] [Q&A]
Hi everyone,

Not sure if this is a bug or not yet, so thought I'd ask the community: is there a (accessbility) reason why the calendar pop-up closes when going back to the current month and `minValue` is set to today? It feels a little jarring, I would have expected the month to show but the previous month button to become disabled, as it is when opened.

https://react-spectrum.adobe.com/react-aria/useDatePicker.html#minimum-and-maximum-values

https://user-images.githubusercontent.com/654103/185946435-feeae428-d16e-404a-a6c6-2986a36d8744.mp4

**Comment by reidbarber** [comment]
Just took a look, and I'm seeing this in Firefox only. Also, it seems to be happening for mouse and keyboard. Not seeing it in the React Spectrum DatePicker components though. Definitely looks like a bug.

**Reply by will-stone** [reply]
Ah yes, I should have said I was using Firefox.
